wear/src/main/res/layout/action_confirm_text.xml

```diff
@@ -11,6 +11,9 @@
         app:boxedEdges="all">
```

Comment:
Let's make the padding smaller to android:padding="5dp" , this will make the text much easier to read on round devices